### ISSUE TITLE
Remove overflow: hidden from .libjass-wrapper

### ISSUE THREAD

**Issue by Arnavion** [Arnavion/libjass#74]
It prevents scrollbars if the video is larger than the viewport.

It exists to hide the .libjass-font-measure divs when they're being used. Need to find another way to hide them, or atleast remove the `overflow: hidden` when they're not being used.

**Comment by XCanG** [comment]
@Arnavion did you try this CSS rule?:
`clip: rect(Y1, X1, Y2, X2) | auto | inherit`
also analogs:
`clip-path: url(#clipping);` link to **svg** file with mask (may be placed in same document at the end)
```
<svg>
  <defs>
    <clipPath id="clipping">
      <rect x="X1" y="Y1" width="W" height="H"/>
    </clipPath>
  </defs>
</svg>
```
`clip-path: polygon(0px 0px, X 0px, X Y, 0px Y);` or directly

`mask-image: url(mask.svg);` or again link to **svg** where black is opaque and white is transparent, 
`mask: url(mask.svg) top left / cover;` will adjust size to element
```
-webkit-mask-box-image: url("border.svg") 100 repeat;
  mask-border: url("border.svg") 100 repeat;
```
another way to masking border content

More info here:
https://developer.mozilla.org/en-US/docs/Web/CSS/clip
https://developer.mozilla.org/en-US/docs/Web/CSS/clip-path
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/rect
https://developer.mozilla.org/en-US/docs/Web/CSS/mask
http://www.html5rocks.com/en/tutorials/masking/adobe/
http://codepen.io/yoksel/full/fsdbu/
https://www.w3.org/TR/css-masking/

**Comment by Arnavion** [comment]
Yes, `clip` is also what I was thinking of.

**Comment by Arnavion** [comment]
Right, now I remember. clips don't work because they affect visibility, not layout, so they don't solve the problem that `overflow: hidden` currently solves (that the font-measure divs don't affect layout of the rest of the page).

**Comment by XCanG** [comment]
Hm, how about rendering on canvas?

**Comment by Arnavion** [comment]
#13 especially the third comment
